core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -472,15 +482,16 @@ func (fm *FluxMonitor) processLogs() {
 			continue
 		}
 
+		ctx, cancel = postgres.DefaultQueryCtx()
+		defer cancel()
 		switch log := broadcast.DecodedLog().(type) {
 		case *flux_aggregator_wrapper.FluxAggregatorNewRound:
-			fm.respondToNewRoundLog(*log)
-			err = broadcast.MarkConsumed()
-
+			fm.respondToNewRoundLog(*log, broadcast)
 		case *flux_aggregator_wrapper.FluxAggregatorAnswerUpdated:
 			fm.respondToAnswerUpdatedLog(*log)
-			err = broadcast.MarkConsumed()
-
+			if err = fm.logBroadcaster.MarkConsumed(fm.db.WithContext(ctx), broadcast); err != nil {
```

Comment:
Again we can probably pull out `db := fm.db.WithContext(ctx)` to make it a bit cleaner